### DISCUSSION TITLE
Icon Request: fa-teapot

### DISCUSSION THREAD

**Discussion by CalumChilds** [FortAwesome/Font-Awesome#21133] [Icon Requests]
A teapot icon could look like this:
![image](https://user-images.githubusercontent.com/25369857/44217521-8f27eb80-a16f-11e8-9d64-4f4b377e048a.png)

It could be used for showing where a cafe is, promoting afternoon tea, etc.

**Comment by keser92300** [comment]
great

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🫖 [U+1FAD6](https://emojipedia.org/emoji/%F0%9F%AB%96/) Teapot

**Comment by WeepingBoil** [comment]
Yes please! ...and for error code 418 !!!!

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
